src/Illuminate/Collections/Collection.php

```diff
@@ -36,11 +36,12 @@ public function __construct($items = [])
      *
      * @param  int  $from
      * @param  int  $to
+     * @param  int|float  $step
      * @return static
      */
-    public static function range($from, $to)
+    public static function range($from, $to, $step = 1)
```

Comment:
The method signature is changed again on these methods.

Reply:
can I target it to 9.x

Reply:
can I target it to 9.x

Reply:
You can try